CMakeLists.txt

```diff
@@ -223,7 +223,8 @@ if(APPLE AND ES_CREATE_BUNDLE)
 	set_target_properties(EndlessSky PROPERTIES
 		MACOSX_BUNDLE_INFO_PLIST "${CMAKE_CURRENT_LIST_DIR}/resources/EndlessSky-Info.plist"
 		XCODE_ATTRIBUTE_ASSETCATALOG_COMPILER_APPICON_NAME endless-sky)
-	set_target_properties(EndlessSky PROPERTIES OUTPUT_NAME "Endless Sky")
+	set(OUTPUT_NAME "Endless Sky")
+	set_target_properties(EndlessSky PROPERTIES OUTPUT_NAME ${OUTPUT_NAME})
```

Comment:
Is this change really necessary? And if it is, shouldn't it be applied to the executables below?

Reply:
I've done this so that any changes to the output name for the Mac OS build here will be automatically propagated to the plist file. I haven't done the same for the Windows or Linux builds because, as far as I know, those names don't need to be set by CMake anywhere else the way the Mac OS build needs it in the plist file. I haven't refused this variable there because the Linux output name is actually different already and the Windows output name isn't necessarily the same as the Mac OS one, it can be different, they just happen to have the same value at the moment.